application/ui/src/features/inspect/toolbar/sources/source-list/utils.ts

```diff
@@ -0,0 +1,3 @@
+export const removeUnderscore = (text: string) => {
```

Comment:
Maybe nitpicky but is this needed? Maybe we can make it inline

Reply:
It’s possible, but I’ll be reusing it in an upcoming PR, so it’s useful to keep it as a reusable function